tests/v2/validation/snapshot/snapshot.go

```diff
@@ -249,31 +254,31 @@ func restoreRKE1(t *testing.T, client *rancher.Client, rke1Snapshot initialSnaps
 	}
 }
 
-func snapshotV2Prov(t *testing.T, client *rancher.Client, podTemplate corev1.PodTemplateSpec, deployment *v1.Deployment, clusterName, clusterID, localClusterID string, etcdRestore *etcdsnapshot.Config, isRKE1 bool) initialSnapshotConfig {
+func snapshotV2Prov(t *testing.T, client *rancher.Client, podTemplate corev1.PodTemplateSpec, deployment *v1.Deployment, clusterName, clusterID, localClusterID string, etcdRestore *etcdsnapshot.Config, isRKE1 bool) (*apisV1.Cluster, initialSnapshotConfig) {
+	if etcdRestore.ReplaceNodes != (etcdsnapshot.ReplaceNodes{}) {
```

Comment:
same question here about the etcdsnapshot object

Reply:
Same comment as above

Reply:
Resolved offline